src/constants.jl

```diff
@@ -15,3 +15,5 @@ import Base.@math_const
 @math_const loghalf -0.6931471805599453094 log(big(0.5))
 
 @math_const sqrthalfπ 1.2533141373155002512 sqrt(big(0.5)*π)
+@math_const sqrt2onπ 0.7978845608028653559 sqrt(big(2.)/π)
```

Comment:
Maybe call this sqrt2overpi? On pi sounds a bit strange to an American English speaker.